src/shared/utils/backoffFetch.ts

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2019 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Defines the maximum backoff.
+const maxBackoff = 512
+
+// Defines the abort controller.
+let controller: AbortController | undefined
+
+// Remakes the controller.
+export const remakeController = () => {
+    if (controller) controller.abort()
```

Comment:
Is there a way to log each request that gets aborted? (Like how we log that we're backing off)